setup.py

```diff
@@ -29,20 +29,20 @@
         "Intended Audience :: Developers",
         "Development Status :: 5 - Production/Stable",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
```

Comment:
I am wondering, I know that 3.6 is not supported anymore, but if Flexmock is still working properly with it, do we need to remove it from the supported versions ? I think it would make more sense to remove it if we upgrade the codebase using features that are not supported in 3.6 ?

Reply:
The problem is that if I don't update the minimum Python version, then many of our development dependencies won't be updated. For example Twisted version 22.10.0 that fixes a security issue requires Python 3.7.1. If someone is still using Python 3.6 when we release a new version, pip would automatically install the latest version that still supports Python 3.6 so everything should still work.

Reply:
Aah, I didn't consider the dev dependencies, so I see ! Feel free to merge :)